pkg/variantregistry/ocp.go

```diff
@@ -115,18 +117,15 @@ ORDER BY j.prowjob_job_name;
 		return nil, errors.Wrap(err, "error querying primary list of all jobs")
 	}
 
-	// TODO: fix release on presubmits
+	var prowJobLastRuns []*prowJobLastRun
 
-	expectedVariants := map[string]map[string]string{}
-
-	count := 0
 	for {
 		// TODO: last run but not necessarily successful, this could be a problem for cluster-data file parsing causing
 		// our churn. We can't flip the query to last success either as we wouldn't have variants for non-passing jobs at all.
 		// Two queries? Use the successful one for cluster-data?
-		jlr := prowJobLastRun{}
-		err := it.Next(&jlr)
-		if err == iterator.Done {
+		jlr := new(prowJobLastRun)
+		err := it.Next(jlr)
```

Comment:
... how would this work without passing a reference to be filled in? (i may be missing something...)

Reply:
Can you clarify what you mean? new creates a pointer to an instance prowJobLastRun{}.   This is mostly existing code, I just prefer to use `new` in for loops to show I'm explicitly creating a new instance in each iteration of the loop.